Firmware_Bootstrap_Loader/faradaybsl/createtiscripter.py

```diff
@@ -78,7 +78,8 @@ def CreateOutputFile(self):
             final_crc = hex(self.CalcTiTxtCrc16(self.section_data_index[i]))
 
             #Script Format
-            script_index_crc = "CRC_CHECK "+"0x"+str(final_addr)+' '+str(final_len)+' '+str(final_crc)
+            script_index_crc = "CRC_CHECK 0x{} {} {}".format(
```

Comment:
Looks benign but alerting @kb1lqd to some changes to the bootstrap loader. I'm tempted to pull this in without running it through firmware loading. You OK with remembering that this was touched if we encounter errors with bootstrap loading in the future?